src/web/topic/components/Surface/TopicToolbar.tsx

```diff
@@ -117,6 +121,22 @@ export const TopicToolbar = () => {
             >
               <Redo />
             </IconButton>
+
+            <Divider orientation="vertical" />
+
+            <IconButton
```

Comment:
can we also wrap this in `{userCanEditTopicData && [component]}` so that it doesn't show up if the user can't edit the topic?

Reply:
it's already gated under https://github.com/amelioro/ameliorate/blob/d876bdcdfd46f9e262b0d82df7777b4846ca96c5/src/web/topic/components/Surface/TopicToolbar.tsx#L91 right?

Reply:
Omg you're right 😅 nice